JsonFlatFileDataStore.Test/SingleItemSecureTests.cs

```diff
@@ -0,0 +1,409 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace JsonFlatFileDataStore.Test
+{
+    public class SingleItemSecureTests
```

Comment:
I'll have to think what is the best way to implement tests so we don't have to duplicate test cases for secure tests.

Maybe initialize DataStore in before each test method and then use same test methods for actual tests.

Reply:
Well it maybe my own brainlagg but right now I cant think anything else then duplicating the test cases.
We could possibly play around with shared data connections, one secure and one plain text.
But for individual test cases you likely want the same operations on both connections.
That way you actually benefit from the information what tests provide by comparing one "apple" to another :)